src/main/java/org/jenkinsci/lib/envinject/EnvInjectAction.java

```diff
@@ -166,11 +166,6 @@ public String transformEntry(String key, String value) {
             return null;
         }
 
-        Job<?, ?> project = build.getParent();
```

Comment:
Is `build.getParent` guaranteed to be non-null? If so the condition in `getEnvVars` shouldn't check if it is null.

Reply:
Javadoc says so...
If you have any concerns, I can revert this change and suppress FindBugs. This is a deprecated code anyway

Reply:
No this is good, I just mean that #11 added code that checks if it is null, so can you remove that check here?